.github/workflows/build.yml

```diff
@@ -12,17 +12,24 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v3
 
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '^1.20.2'
+
+      - name: Set up Docker
+        uses: docker/setup-buildx-action@v2
+
       - name: Generate tag name
         id: tag
         run: |
-          IMAGE_TAG="ghcr.io/grasse-oss/cron-set-controller:"${GITHUB_REF#refs/tags/v}"
+          # IMAGE_TAG="ghcr.io/grasse-oss/cron-set-controller:"${GITHUB_REF#refs/tags/v}"
+          IMAGE_TAG="ghcr.io/grasse-oss/cron-set-controller:0.0.2"
```

Comment:
Is there a reason why the tag should be fixed at 0.0.2?

Reply:
It was temporary added for testing.